bin/www

```diff
@@ -4,7 +4,7 @@
  * Module dependencies.
  */
 
-var app = require('../server/index');
+var app = require('../server/app');
```

Comment:
Can we move this app.js file back up to the root?  Still not sure why/how this got moved.